apps/server/src/middleware/exception-logger.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ArgumentsHost,
+  Catch,
+  ExceptionFilter,
+  HttpException,
+  Logger,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+
+import { REQUEST_ID } from '../constants';
+
+@Catch(HttpException)
```

Comment:
what about other errors

Reply:
we don't have rpc request, for ws request I suggest don't catch them, make socketIO handle it